stories/Overlays/Alert.stories.jsx

```diff
@@ -38,6 +38,28 @@ export const Default = () => {
   );
 };
 
+export const AlertLoading = () => {
```

Comment:
We don't have to add this story, since we are going to remove the loading prop from alert and modal. Please revert the changes related to this story.